.release/buildspec.yml

```diff
@@ -3,7 +3,7 @@ version: 0.2
 phases:
   install:
     runtime-versions:
-      golang: 1.15
+      golang: 1.16
```

Comment:
Hi unfortunately this is not yet available with CodeBuild: https://docs.aws.amazon.com/codebuild/latest/userguide/runtime-versions.html 

This is why we haven't upgraded to go1.16 yet 😭 

We would need to do something like this to accept the changes: https://github.com/aws/aws-codebuild-docker-images/issues/425#issuecomment-861921069

Reply:
Interesting, after four months, and codebuild still does not support go1.16 😞